webgpu.h

```diff
@@ -566,6 +571,41 @@ typedef struct WGPUStencilStateFaceDescriptor {
     WGPUStencilOperation passOp;
 } WGPUStencilStateFaceDescriptor;
 
+typedef struct WGPUSurfaceDescriptor {
+    void const * nextInChain;
+    char const * label;
+} WGPUSurfaceDescriptor;
+
+typedef struct WGPUSurfaceDescriptorFromMetalLayer {
```

Comment:
Any reason this is from MetalLayer and not more generally from a CALayer? It seems like wgpu implementations could create a child CAMetalLayer if needed so that users don't need to know about Metal at all.

Reply:
Ack, we can always add another surface type if someone really really needs to pass a CALayer.